src/libs/ReportUtils.ts

```diff
@@ -545,6 +546,13 @@ function isReportApproved(report: OnyxEntry<Report>): boolean {
     return report?.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && report?.statusNum === CONST.REPORT.STATUS.APPROVED;
 }
 
+/**
+ * Checks if the supplied report is in Open state and status.
+ */
+function isReportDraft(report: OnyxEntry<Report>): boolean {
```

Comment:
NAB: I think this is slightly more explicit about the type. 
```suggestion
function isDraftExpenseReport(report: OnyxEntry<Report>): boolean {
```